### ISSUE TITLE
Photo of each blog in all blog lists page

### ISSUE THREAD

**Issue by rushjam** [rushjam/portfolio-blog#2]

**Comment by goutham03062001** [comment]
Hello is this isssue still opened? I would like to contribute in this project.

**Comment by rushjam** [comment]
Yes, the issue is still open. However, initial work is already started. So, it is no more just a **good first issue**. But yes, you can definitely contribute to the current issue. I will update the description for the same for understanding the issue in detail.
